### ISSUE TITLE
Low contrast in patron brief view 

### ISSUE THREAD

**Issue by kfischer** [jajm/koha-staff-interface-redesign#19]
In the patron preview there is an example of bad contrast with grey font on grey background:

![grafik](https://user-images.githubusercontent.com/425526/189328831-f670c345-516a-4f3d-b8e3-b8ace5031ecd.png)

The "no email stored" text is too hard to read.

**Comment by jajm** [comment]
I changed the "very light grey" to a color that pass WCAG AA test, and also changed the other text color to black, otherwise the two colors were too similar

**Comment by jajm** [comment]
![image](https://user-images.githubusercontent.com/306836/189848766-338b0ab5-e6c0-4449-979f-9026f3cded82.png)

**Comment by kfischer** [comment]
Hi Julian, looks much better! Also great to know we are passing WCAG now here!
